README.md

```diff
@@ -378,17 +411,26 @@ map.pm.toggleGlobalRemovalMode();
 
 The following methods are available on `map.pm`:
 
-| Method                    | Returns   | Description                                                              |
-| :------------------------ | :-------- | :----------------------------------------------------------------------- |
-| toggleGlobalRemovalMode() | -         | Toggles global removal mode.                                             |
-| globalRemovalEnabled()    | `Boolean` | Returns `true` if global removal mode is enabled. `false` when disabled. |
+| Method                     | Returns   | Description                                                              |
+| :------------------------- | :-------- | :----------------------------------------------------------------------- |
+| enableGlobalRemovalMode()  | -         | Enables global drag mode.                                                |
+| disableGlobalRemovalMode() | -         | Disables global drag mode.                                               |
+| toggleGlobalRemovalMode()  | -         | Toggles global removal mode.                                             |
+| globalRemovalEnabled()     | `Boolean` | Returns `true` if global removal mode is enabled. `false` when disabled. |
+
+The following events are available on a layer instance:
+
+| Event       | Params | Description                                              | Output  |
+| :---------- | :----- | :------------------------------------------------------- | :------ |
+| pm:remove   | `e`    | Fired when a layer is removed via Removal Mode           | `layer` |
 
 The following events are available on a map instance:
 
-| Event       | Params | Description                                              |
-| :---------- | :----- | :------------------------------------------------------- |
-| pm:remove   | `e`    | Fired when a layer is removed via Removal Mode           |
-| layerremove | `e`    | Standard Leaflet event. Fired when any layer is removed. |
+| Event                         | Params | Description                                              | Output            |
```

Comment:
This should theoretically add a fourth column to the table, right? Somehow, VSCode is not showing this to me but I see the code in the README 🤔

![image](https://user-images.githubusercontent.com/2399810/82421198-faa79700-9a80-11ea-815c-81fbfa07426c.png)

Does "Output" has another effect that I'm not aware of?

Reply:
In my editor it looks well and in [stackedit](https://stackedit.io/app#) it looks also good. I don't know what I should change 🤷‍♂️

Output is the payload